### ISSUE TITLE
Occasional test failure

### ISSUE THREAD

**Issue by nijel** [WeblateOrg/weblate#4410]
**Describe the bug**

Sometimes the test_duplicate_checks tests fail:

```
======================================================================
FAIL: test_check_multiple_duplicated_tokens (weblate.checks.tests.test_duplicate_checks.DuplicateCheckTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/tmp.YvzGW7R1Ap/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/weblate/checks/tests/test_duplicate_checks.py", line 50, in test_check_multiple_duplicated_tokens
    self.assertTrue(self._run_check("I have two two lemons lemons"))
AssertionError: False is not true

======================================================================
FAIL: test_check_single_duplicated_token (weblate.checks.tests.test_duplicate_checks.DuplicateCheckTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/tmp.YvzGW7R1Ap/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/weblate/checks/tests/test_duplicate_checks.py", line 47, in test_check_single_duplicated_token
    self.assertTrue(self._run_check("I have two two lemons"))
AssertionError: False is not true

======================================================================
FAIL: test_description (weblate.checks.tests.test_duplicate_checks.DuplicateCheckTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/tmp.YvzGW7R1Ap/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/weblate/checks/tests/test_duplicate_checks.py", line 71, in test_description
    "Text contains the same word twice in a row: lemons, two",
AssertionError: 'Text contains the same word twice in a row: ' != 'Text contains the same word twice in a row: lemons, two'

----------------------------------------------------------------------
```

All three tests fail at same time.

**To Reproduce**

I have no clue how to trigger this. It sometimes happens in the CI, for example in https://github.com/WeblateOrg/weblate/runs/1050994943 or https://github.com/WeblateOrg/weblate/runs/1133769829

**Expected behavior**

These tests should pass every time.

**Comment by nijel** [comment]
Still happens: https://github.com/WeblateOrg/weblate/runs/1274808665?check_suite_focus=true

**Comment by nijel** [comment]
Seems like this was finally fixed by 7b05107.
